lib/YaST/Bootloader/BootloaderSettingsController.pm

```diff
@@ -79,6 +80,15 @@ sub write_to_partition {
     $self->get_boot_code_options_page->press_next();
 }
 
+sub disable_grub_timeout_right_key {
```

Comment:
the existence of this function using testapi worth a mention to the bug and why is there in the form of a comment? wdyt?